Dockerfile

```diff
@@ -3,7 +3,7 @@ LABEL MAINTAINER="BJ Dierkes <derks@datafolklabs.com>"
 ENV PS1="\[\e[0;33m\]|> cement <| \[\e[1;35m\]\W\[\e[0m\] \[\e[0m\]# "
 WORKDIR /src
 COPY . /src
-RUN python setup.py install \
+RUN pip install . \
```

Comment:
I think this is an integration test - but  I didn't quite understand the need to move `.` to `src`